### PR TITLE
Support Swift 5.10

### PR DESCRIPTION
**Pull request by marcprux**
This PR adds support for Swift 5.10: https://www.swift.org/blog/swift-5.10-released/

### CODE REVIEW

**Comment by grdsdev** [comment]
Hi @marcprux this going to be very helpful, there is any thing I can do to help this getting merged?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-swift",
-  "version": "1.26.0",
+  "version": "0.1.0",
```
[the remark this answers is not in the document]
Comment (marcprux) [reply package.json:3]: > Looking at other PRs adding the latest swift version, this change doesn't seem to be necessary, right?

TBH, I don't know. That was automatically added by "GitHub Actions (Release Drafter)".

**Comment by marcprux** [comment]
> Hi @marcprux this going to be very helpful, there is any thing I can do to help this getting merged?

@fwal might be able to comment on the expected timeframe or any further review that is needed.

**Review by fwal — APPROVED** [review]
Thanks! Getting it out now!

**Pull request merged**